docs/ldml/tr35-keyboards.md

```diff
@@ -770,7 +776,7 @@ _Attribute:_ `to`
 
 _Attribute:_ `transform="no"` (optional)
 
-> The `transform` attribute is used to define a key that does not participate in a transform (until the next keystroke). This attribute value must be `no` if the attribute is present.
+> The `transform` attribute is used to define a key that does not participate in a transform (until the next keystroke). This attribute value must be `"no"` if the attribute is present.
```

Comment:
The **value** is the part in quotes, and does not include the quotes. X='no' is the same as X="no"

Reply:
I'm not sure what you are requesting then, could you use the suggest feature?